Source/MSBuildTools.Unity/Packages/com.microsoft.msbuildforunity/Editor/ProjectGenerator/Scripts/Templates/TemplateFiles.cs

```diff
@@ -33,6 +33,7 @@ public class TemplateFiles
         private const string SpecifcPlatformPropsTemplateRegex = @"[a-zA-Z]+\.[a-zA-Z]+\.[a-zA-Z0-9]*\.props.template";
         private const string PluginMetaFileTemplateRegex = @"Plugin\.([a-zA-Z]*)\.meta.template";
         private const string BuildProjectsTemplateName = "BuildProjects.proj.template";
+        private const string NugetConfigFileName = "NuGet.config";
```

Comment:
You also want to add the Property for NuGetConfigFile, and set the property as done below by something like "BuildProjectsTemplateName"

Reply:
Fixed